src/pages/HistoryPage/EpochForm.tsx

```diff
@@ -394,7 +654,7 @@ const EpochForm = ({
             <Button
               color="primary"
               type="submit"
-              disabled={submitting || !isDirty || !isEmpty(errors)}
+              disabled={submitting || !isEmpty(errors)}
               onClick={handleSubmit(onSubmit)}
             >
               {submitting ? 'Saving...' : 'Save'}
```

Comment:
This doesn't reset after we receive an error

Reply:
it does now